packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.test.tsx

```diff
@@ -101,6 +101,30 @@ describe('CommandBar', () => {
     expect(document.querySelector('.SubMenuClass')).toBeDefined();
   });
 
+  it('passes event and item to button onClick callbacks', () => {
+    let testValue: string | undefined = '';
+    const commandBar = mount(
+      <CommandBar
+        items={[
+          {
+            text: 'TestText 1',
+            key: 'TestKey1',
+            className: 'MenuItem',
+            onClick: (ev, item) => {
+              testValue = ev && item && item.text;
+            }
```

Comment:
could add a test for passing data as well? (If the item has the data property, it'll pass in the data with the item)